Mathlib/Analysis/Normed/Module/Dual.lean

```diff
@@ -56,7 +56,7 @@ variable (E : Type*) [SeminormedAddCommGroup E] [NormedSpace 𝕜 E]
 variable (F : Type*) [NormedAddCommGroup F] [NormedSpace 𝕜 F]
 
 /-- The topological dual of a seminormed space `E`. -/
-abbrev Dual : Type _ := E →L[𝕜] 𝕜
+abbrev Dual : Type _ := TopologicalSpace.Dual 𝕜 E
```

Comment:
I would be tempted to omit this `abbrev` entirely, and switch to `StrongDual` (or whatever name we choose), but someone else (or multiple people) should agree before we make that jump.

Reply:
I did wonder about `alias Dual := StrongDual` instead, but that seems not to work.

Reply:
[#mathlib4 > Name for the continuous linear functionals of a topological](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/Name.20for.20the.20continuous.20linear.20functionals.20of.20a.20topological)

Reply:
Presumably we're also going to need to rename:

inclusionInDoubleDual -> inclusionInDoubleStrongDual
dual_def -> strongDual_def
dualPairing -> strongDualPairing
dualPairing_apply -> strongDualPairing_apply
norm_le_dual_bound -> norm_le_strongdual_bound
eq_zero_of_forall_dual_eq_zero -> eq_zero_of_forall_strongdual_eq_zero
eq_iff_forall_dual_eq -> eq_iff_forall_strongdual_eq
inclusionInDoubleDualLi -> inclusionInDoubleStrongDualLi

etc.

Reply:
My original plan was to not rename at all; `Module` is actually a semi-module, `MonoidHom` doesn't actually require associativity, etc.

Reply:
@j-loreaux I've started looking at the first one `NormedSpace.apply` which depends on `ContinuousLinearMap.flip` which currently presumes seminormed spaces. I think making that work more generally requires hypocontinuous bilinear mappings (Bourbaki TVS III, §5.3). Is that right?

Reply:
I've moved these into a separate file `Topology/Algebra/Module/StrongDual` and changed the namespace to `StrongDual`. I renamed `StrongDual.dualPairing` to `_root_.strongDualPairing`. I left `StrongDual.polar` and `StrongDual.polarSubmodule` in the namespace to disambiguate from the other uses of `polar`.